client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -249,9 +220,22 @@ class ActivityLogItem extends Component {
 	}
 
 	renderHeader() {
+		const { actor = {} } = this.props.log;
+		const {
```

Comment:
This sounds like a reasonable approach if the actor has a couple of attributes.

If it eventually grows to be something like the `site` object, it would probably make sense to pass the object instead.

Another case where it wouldn't be reasonable to pass attributes specifically as props would be if the actor object can vary in terms of what attributes might contain. Not sure if that might become a real case with activity actors.